Source/Layout/ASLayout.mm

```diff
@@ -76,6 +76,13 @@ @implementation ASLayout
 
 @dynamic frame, type;
 
+static BOOL static_retainsSublayoutLayoutElements = NO;
```

Comment:
That is because `ASLayout` also has an instance variable called `retainSublayoutLayoutElements` [here](https://github.com/facebookarchive/AsyncDisplayKit/blob/master/Source/Layout/ASLayout.mm#L57).